content/kubermatic/master/_index.en.md

```diff
@@ -9,30 +9,73 @@ date = 2019-04-27T16:06:34+02:00
 
 ## What is Kubermatic Kubernetes Platform (KKP)?
 
-KKP is one enterprise Kubernetes management platform for any infrastructure. KKP automates thousands of Kubernetes clusters across multi-cloud, on-prem and edge with unparalleled density and resilience. Deploy, manage and run multiple Kubernetes clusters with our production-proven platform on your preferred infrastructure.
+Kubermatic Kubernetes Platform (KKP) is a Kubernetes management platform that helps address the operational and security challenges of enterprise customers seeking to run Kubernetes at scale. KKP automates deployment and operations of hundreds or thousands of Kubernetes clusters across hybrid-cloud, multi-cloud and edge environments while enabling DevOps teams with a self-service developer and operations portal.
+
 
 KKP is directly integrated with leading cloud providers such as [Amazon Web Services](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/aws/aws/), [Azure](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/azure/azure/), DigitalOcean, [Google Compute Engine](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/google_cloud/gcp/), Hetzner, OpenStack, Packet and [VMware vSphere](https://docs.kubermatic.com/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/vsphere/vsphere/) as well as any provider offering Ubuntu 16.04 or greater, even in your own datacenter.
 
-## Want to start with KKP in a GitOps way?
-Check the [Start with KKP]({{< ref "./installation/startio/" >}}) for more details.
 
 ## Features
 
-{{% notice tip %}}
-For latest updates follow us on Twitter [@Kubermatic](https://twitter.com/Kubermatic)
-{{% /notice %}}
+#### Powerful & intuitive dashboard to visualize your Kubernetes deployment
+Manage your projects and clusters with the KKP dashboard. Scale your cluster by adding and removing nodes in just a few clicks. As an Admin, the dashboard also allows you to customize the theme and disable theming options for other users.
```

Comment:
> As an Admin, the dashboard also allows you to customize the theme and disable theming options for other users.

I'm not sure is this worth highlighting on the main page. This is a nice feature, but I think we have many other features that are much more compelling.

Reply:
It is needed for SEO/having a complete overview